src/Gateway.php

```diff
@@ -56,4 +56,19 @@ public function completePurchase(array $parameters = array())
     {
         return $this->createRequest('\Omnipay\Netaxept\Message\CompletePurchaseRequest', $parameters);
     }
+
+    public function capture(array $parameters = array())
+    {
+        return $this->createRequest('\Omnipay\Netaxept\Message\CaptureRequest', $parameters);
+    }
+
+    public function annul(array $parameters = array())
```

Comment:
In order to stay in line with the rest of Omnipay, this would need to be `void()`